Yosemite/Yosemite/Model/Payments/WCPayAccount+PaymentGatewayAccount.swift

```diff
@@ -0,0 +1,21 @@
+import Hardware
+
+public extension WCPayAccount {
+    /// Maps a WCPayAccount into the PaymentGatewayAccount struct
+    ///
+    func toPaymentGatewayAccount(siteID: Int64) -> PaymentGatewayAccount { // TODO can we add siteID to WCPayAccount?
```

Comment:
WCPayAccount doesn't have a siteID right now. Since we aren't storing it, I'm not sure it is appropriate to add it to it, and that this `toPaymentGatewayAccount` doing it makes sense, but I am open to y'alls thoughts

Reply:
It seems to me like there should be a relationship between WCPayAccount and a site, right? It's something we can add later. I guess.